simtools/applications/simulate_prod.py

```diff
@@ -221,72 +168,25 @@ def main():  # noqa: D103
     logger = logging.getLogger()
     logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
 
-    try:
-        config_data = gen.collect_data_from_file_or_dict(
-            file_name=args_dict["production_config"], in_dict=None
-        )
-    except FileNotFoundError:
-        logger.error(
-            f"Error loading simulation configuration file from {args_dict['production_config']}"
-        )
-        raise
-
-    # Overwrite default and optional settings
-    config_data["showers"]["run_list"] = args_dict["run"] + args_dict["start_run"]
-    config_data["showers"]["primary"] = args_dict["primary"]
-    config_data["common"]["site"] = args_dict["site"]
-    config_data["common"]["zenith"] = args_dict["zenith_angle"]
-    config_data["common"]["phi"] = args_dict["azimuth_angle"]
-    label = config_data["common"].pop("label", "test-production")
-    config_data["common"]["data_directory"] = Path(args_dict["data_directory"]) / label
-
-    if args_dict["nshow"] is not None:
-        config_data["showers"]["nshow"] = args_dict["nshow"]
-    if args_dict["label"] is not None:
-        label = args_dict["label"]
-
     simulator = Simulator(
```

Comment:
The above comment is related to this: We don't want to keep the option to set the parameters by hand and overwrite the default settings?

Reply:
Same here, this functionality is simply moved to the command_line module (and therefore generalized to avoid duplication).